### ISSUE TITLE
class LpConstraintVar is not documented

### ISSUE THREAD

**Issue by MLopez-Ibanez** [coin-or/pulp#461]
Details for the issue
--------------------

#### What did you do?

The documentation does not include the information regarding `LpConstraintVar`:

https://coin-or.github.io/pulp/technical/pulp.html?highlight=lpconstraintvar

#### What did you expect to see?

At minimum, a description of the __init__ and various parameters such as "rhs", "sense", etc.

#### What did you see instead?

Nothing.

Useful extra information
-------------------------

#### Did you also

- [X ] Tried out the latest github version: https://github.com/coin-or/pulp
- [X] Searched for an existing similar issue: https://github.com/coin-or/pulp/issues?utf8=%E2%9C%93&q=is%3Aissue%20

**Comment by pchtsp** [comment]
Hello! You're right, we do not have that documented in pulp. Personally, although I more or less understand the purpose, I've never used `LpConstraintVar` so I'm not the best candidate. @stumitchell I'm guessing you know a little more? @MLopez-Ibanez maybe you want to volunteer to do some documentation and share ir via a PR :D  ?

**Comment by MLopez-Ibanez** [comment]
Hi @pchtsp ! Actually, this was my first roadblock towards understanding how to do column generation with Pulp.

**Comment by pchtsp** [comment]
Have you taken a look at the column generation examples in PuLP?
https://github.com/coin-or/pulp/blob/master/examples/CGcolumnwise.py
https://github.com/coin-or/pulp/blob/master/examples/CG.py
The first one uses `LpConstraintVar`.

I'm sorry I can't be of more help.

**Comment by MLopez-Ibanez** [comment]
> Have you taken a look at the column generation examples in PuLP? https://github.com/coin-or/pulp/blob/master/examples/CGcolumnwise.py https://github.com/coin-or/pulp/blob/master/examples/CG.py The first one uses `LpConstraintVar`.

Yes, but I still do not understand that `LpConstraintVar` is doing and how to properly call it. If I knew I could document it.

**Comment by pchtsp** [comment]
Well, I've now dropped support for LpConstraintVar in the latest upcoming version (4.0). If you want to re-implement some funcionality on this topic is of course welcome!
